app/build.gradle

```diff
@@ -1,15 +1,15 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    compileSdkVersion 33
```

Comment:
@pbulka neviem co to moze sposobit problem ale v trucku pouzivame 32

Reply:
Nemalo by.